ideascube/serveradmin/tests/test_catalog_command.py

```diff
@@ -90,10 +90,13 @@ def test_cannot_add_duplicate_remote(tmpdir, settings, monkeypatch):
 
     old_mtime = tmpdir.join('remotes', 'foo.yml').mtime()
 
+    call_command(
```

Comment:
Ah, got it :)
Can you add a quick comment for future me that will forget that again? :)
Something like: `# Trying to add the same remote with the same URL should not fail`.